src/pages/agent/tests-to-run/tests-to-run-header/get-suggested-tests-modal/get-suggested-tests-modal.tsx

```diff
@@ -35,6 +31,8 @@ export const GetSuggestedTestsModal = ({ agentType }: Props) => {
     return () => clearTimeout(timeout);
   }, [copied]);
   const closeModal = useCloseModal("/get-suggested-tests");
+  const agentType = "Agent";
```

Comment:
agent type = Agent | Group

Reply:
Why put it in a constant?

Reply:
- This component is used in the agent part of t2c only.  
- Because the string is needed in two places, but I agree that I can use a literal